templates/artist/artist_detail.html

```diff
@@ -110,9 +117,11 @@ <h1 class="header-font">{{ artist.name }}</h1>
             <hr>
         </div>
         <!-- Music -->
-        <h2>Music</h2>
-        {% for playlist in artist.soundcloudplaylist_set.all %}
-            <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
+        <h2>MUSIC</h2>
+        <div class="extra-padding">
+            {% for playlist in artist.soundcloudplaylist_set.all %}
+                <iframe width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{ playlist.playlist }}&color=ff5500"></iframe>
+        </div>
```

Comment:
This `<div>` should come after the `{% for %}` loop closes. Otherwise, we're creating a `</div>` for every sound cloud playlist the artist has (can be multiple).